app/classifier/tasks/highlighter/index.jsx

```diff
@@ -36,6 +37,23 @@ export default class Highlighter extends React.Component {
     selection.collapseToEnd();
   }
 
+  getOffset(selection) {
+    const { anchorNode } = selection;
+    const { parentNode } = anchorNode;
+    let start = 0;
+    let counting = true;
+    parentNode.childNodes.forEach((node) => {
+      // ignore comments (nodeType 8) when calculating offset distance
+      if (counting && node.nodeType !== 8 && node !== anchorNode) {
+        start += node.textContent.length;
+      }
+      if (node === anchorNode) {
+        counting = false;
```

Comment:
This is a bit of a hack to get around `NodeList` not being an array, so we can't use `break` to get out of the loop once we find the node we're working on.